### DISCUSSION TITLE
Hide operator for combo box in filters

### DISCUSSION THREAD

**Discussion by cirko97** [microsoft-search/pnp-modern-search#2940] [General]
Hello,

I'm search for documents on my site with some columns to filter etc. 

I'm intersted if there's any way to hide AND/OR (operator) option in combo box inside Filters, so the end user doesn't see that option and can't change whatever I set in edit mode?

**Comment by kasperbolarsen** [comment]
As far as ensuring that the users doesn't change your settings: isn't that solved by breaking the inheritance on the page and only allow read permissions for everybody but the site collection administrators?
